docs/usage.md

```diff
@@ -223,6 +223,19 @@ const sequelize = new Sequelize('database', 'username', 'password', {
 })
 ```
 
+### ORACLE
+
+The library for Oracle is`oracledb@^1.11.0` You'll just need to define the dialect:
```

Comment:
It would be better to use 1.13.1

Reply:
the ^ in front of the version number indicates version from 1.11.0 to latest. In package.json it's the same, so the last version will be downloaded

Reply:
If the latest is always used, then you may as well update the text to 1.13.1!  Various bugs were fixed, LOB support was improved in versions after 1.11.0, and pool pinging were introduced.  I'd like to see everyone update to 1.13.1 :)